src/goose/brokers/sqs/sqs_requests.clj

```diff
@@ -0,0 +1,12 @@
+(ns goose.brokers.sqs.sqs-requests
+  (:require
+   [cognitect.aws.client.api :as aws]
+   [clojure.data.json :as json]))
+
+(defn enqueue
+  "Enqueues a job for immediate execution."
+  [client queue-url job]
+  (aws/invoke client {:op      :SendMessage
```

Comment:
Is there an underlying connection pool that is utilized by aws/invoke? Could we have access to this pool and pass in explicitly as a dependency?

Reply:
No as of now it does not expose a direct connection pool. We can customize it though https://github.com/cognitect-labs/aws-api?tab=readme-ov-file#overriding-the-http-client